Update windows-autopilot-hybrid.md (#2591)
MicrosoftDocs/memdocs: memdocs/autopilot/windows-autopilot-hybrid.md

    - **Name**: Enter a descriptive name for the new profile.
    - **Description**: Enter a description for the profile.
    - **Platform**: Select **Windows 10 and later**.

>     - **Profile type**: Select **Tempplates**, Choose the Template name **Domain Join** and click on **Create**

Reviewer: ```suggestion
    - **Profile type**: Select **Templates**, choose the template name **Domain Join** and select **Create**.
```

> 3. Enter the **Name**, **Description** and Click on **Next** 

Reviewer: ```suggestion
3. Enter the **Name** and **Description** and select **Next**. 
```

> 5. Provide a **Computer name prefix**, **Domain name**.

Reviewer: ```suggestion
5. Provide a **Computer name prefix** and **Domain name**.
```
